### PR TITLE
Clio 43409 th dropdown 1.6

### PR DESCRIPTION
**Pull request by frankieyan**

### CODE REVIEW

File: src/lib/thDropdown/thDropdown.directive.coffee [hunk 1]
```diff
@@ -122,7 +126,7 @@ angular.module("ThemisComponents")
         adjustedRight = bodyRightPosition - buttonRect.right
 
         # dropdown menu list defaults to right alignment to button
-        # if the button is left aligned let's set menu left align
+        # if the button is left aligned let"s set menu left align
```
Comment (jhnbrnn) [inline src/lib/thDropdown/thDropdown.directive.coffee]: the tiniest nitpick, but this doesn't need to be a double quote 😉

File: src/lib/thDropdown/thDropdown.directive.coffee [hunk 2]
```diff
@@ -97,20 +102,19 @@ angular.module("ThemisComponents")
           else
             @processedItems.push item
 
-      processList()
       return
 
     link: (scope, elem, attr) ->
       # set button to disabled if attr passed
-      elem.find("a").attr('disabled', 'disabled') if attr.disabled?
+      elem.find("button").attr("disabled", "disabled") if attr.disabled?
```
Comment (jhnbrnn) [inline src/lib/thDropdown/thDropdown.directive.coffee]: This child element has already been changed to a button in the template, I'm guessing?

**Comment by jhnbrnn** [comment]
One tiny thing and one question, but looks good to me 👍

File: src/lib/thDropdown/thDropdown.directive.coffee (hunk 1 above)
Comment (frankieyan) [inline src/lib/thDropdown/thDropdown.directive.coffee]: Thought I'd be smart and use find/replace but nooo

File: src/lib/thDropdown/thDropdown.directive.coffee (hunk 2 above)
Comment (frankieyan) [inline src/lib/thDropdown/thDropdown.directive.coffee]: Yep, but going to remove this line altogether as the disabled attribute is already being set on the controller above. This should probably be using an `ng-disabled` directive instead on the template - but I don't want to introduce too many changes at this point.

**Comment by jhnbrnn** [comment]
:shipit:

**Pull request merged**